src/Typed/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG
+
+## 1.0
```

Comment:
You can delete this file. We'll bootstrap it later, and the initial release would actually be 2.2 :)

Reply:
OK so let's remove it.